pages/common/rg.md

```diff
@@ -25,3 +25,7 @@
 - Only list matched files -- useful when piping to other commands:
 
 `rg --files-with-matches {{pattern}}`
+
+- Invert matching. Show lines that do not match the given pattern:
+
+`rg -v {{pattern}}`
```

Comment:
There isn't a long option here, is there?

Reply:
Yes, there is. It is `--invert-match`. Should we use this instead of the short one?